helpers.ts

```diff
@@ -52,6 +52,17 @@ export function trackDownloadProgress(destinationStream: NodeJS.WritableStream,
 	return progressStream;
 }
 
+export function isBuildFromCLI(item: string) {
+	return item.match(/.*\.aar/) ||
```

Comment:
this regex will match strings like:
* filename.aar
* .aar
* test.aar.backup

Is this the desired behavior?

Reply:
yes